src/PolyhedralGeometry/PolyhedralComplex/standard_constructions.jl

```diff
@@ -74,3 +74,99 @@ function k_skeleton(PC::PolyhedralComplex{T}, k::Int) where {T<:scalar_types}
   )
   return PolyhedralComplex{T}(ksk, coefficient_field(PC))
 end
+
+###############################################################################
+## Scalar multiplication
+###############################################################################
+
+# helper functions to ensure that point and ray vectors scale differently:
+# - point vectors represent points in space and should be scaled normally
+# - ray vectors represent directions and they should only be scaled by 0 or +/-1
+function scale(
+  u::PointVector{<:scalar_types_extended}, c::scalar_types_extended
+)
+  return u * c
+end
+function scale(
+  u::RayVector{<:scalar_types_extended}, c::scalar_types_extended
+)
+  if is_positive(c)
+    return u
+  end
+  if is_negative(c)
+    return -u
+  end
+  return 0*u
+end
```

Comment:
Is `sign` available for all `scalar_types_extended`? If so you could shorten it a bit:
```suggestion
function scale(
  u::RayVector{<:scalar_types_extended}, c::scalar_types_extended
)
  if is_positive(c)
    return u
  else
  	return sign(c)*u
  end
end
```
It could also be shortened more, but maybe it gets slower then?

Reply:
Sorry, a train ride interrupted my review, so this is outdated.